### PR TITLE
use keyword type for url - text instead of string for other field

### PR DESCRIPTION
**Pull request by AdriwanKenoby**
Hi @macbre ,

I have correct some bug:

- type string replace by text
- url must be of type keyword and indexed

Accordingly to [https://www.elastic.co/blog/strings-are-dead-long-live-strings](https://www.elastic.co/blog/strings-are-dead-long-live-strings)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
   "homepage": "https://github.com/macbre/phantomas-reporter-elasticsearch#readme",
   "dependencies": {
     "debug": "^2.2.0",
-    "elasticsearch": "^14.1.0"
+    "elasticsearch": "^14.1.0",
+    "public-ip": "^2.4.0"
```
Comment (macbre) [inline package.json]: Is this dependency used here?

**Comment by macbre** [comment]
Please keep the tabs as indentation. It would be way easier to read the diff :) Thanks!

File: package.json (hunk 1 above)
Comment (AdriwanKenoby) [reply package.json]: no this dependency is unused here I just forget to remove it

**Comment by AdriwanKenoby** [comment]
sorry for the tabs indent

**Pull request merged**